### ISSUE TITLE
Transmart Logo broken on Login Page

### ISSUE THREAD

**Issue by BenjaminHCCarr** [i2b2-tranSMART/i2b2transmart-quick-start#2]
On the login page, the Trasmart Logo is missing:
![transmart_login](https://user-images.githubusercontent.com/8225968/39529046-bd0ec4f8-4df3-11e8-999b-b63141977814.png)

When you try to open on a new window you receive:
```
HTTP Status 404 - "/images/index.gsp" not found.
type Status report

message "/images/index.gsp" not found.

description The requested resource is not available.

Apache Tomcat/8.0.50
```

This is a `gsp` issue similar to #1, possible similar cause.

**Comment by recursethis** [comment]
Should be resolved with latest build.

Pull the latest from master, and run the following:
```bash
$ docker-compose pull transmart
$ docker-compose up -d transmart
```

**Comment by BenjaminHCCarr** [comment]
Thanks @recursethis this fixes the logo issue, but doesn't affect #1
